### ISSUE TITLE
"private" methods not quite handled correctly by show

### ISSUE THREAD

**Issue by bruceravel** [xraypy/xraylarch#119]
I am working on the infrastructure for the tests of the effect of self-consistency that I presented last week at the XAFS conference.  Specifically, I am replacing a hodge-podge of python, larch, shell, and perl scripts with one sensible larch plugin.

It registers itself like this:
```python
def registerLarchPlugin(): # must have a function with this name!
    return ('fefftest', { 'ft': ft })
```
that is, it does not register itself into one of the standard groups (like `_xafs`).

When I do a show, this happens:

```
 == Feff Test Group: bromoadamantane: 24 symbols ==
  _FeffTestGroup__cull: <bound method FeffTestGroup.__cull of <Feff Test Group: bromoadamantane>>
  _FeffTestGroup__target: <bound method FeffTestGroup.__target of <Feff Test Group: bromoadamantane>>
 .
  .
   .
```

The motif of ignoring methods that start with a dunder does not happen because the name of the method is prepended with the Group name.

Obviously, this isn't a disaster, but it is a bit irksome.  I don't quite understand Larch's symbol table well enough to propose a good fix for `plugin/std/show.py`.

**Comment by newville** [comment]
agree this is irritating, and thought it was handled better.  Will investigate.

**Comment by newville** [comment]
fixed in 1d2c13297be

**Comment by bruceravel** [comment]
Neat! :+1:
